apps/generator/lib/generator.js

```diff
@@ -867,34 +887,43 @@ class Generator {
   async generateFile(asyncapiDocument, fileName, baseDir) {
     const sourceFile = path.resolve(baseDir, fileName);
     const relativeSourceFile = path.relative(this.templateContentDir, sourceFile);
+    const relativeSourceDirectory = relativeSourceFile.split(path.sep)[0] || '.';
+  
     const targetFile = path.resolve(this.targetDir, this.maybeRenameSourceFile(relativeSourceFile));
     const relativeTargetFile = path.relative(this.targetDir, targetFile);
-
+    let shouldGenerate = true;
+  
     if (shouldIgnoreFile(relativeSourceFile)) return;
+    
+    if (!(await this.shouldOverwriteFile(relativeTargetFile))) return;
 
-    const shouldOverwriteFile = await this.shouldOverwriteFile(relativeTargetFile);
-    if (!shouldOverwriteFile) return;
-
+    // The conditionalFiles configuration will deprecate soon 
```

Comment:
it becomes deprecated with this PR, and soon will be removed

Reply:
thanks done